### PR TITLE
Refine loader.qml UI and add digital-fat-bwoy-slim watchface

### PR DESCRIPTION
**Pull request by eLtMosen**
### Add digital-fat-bwoy-slim watchface with NASDAQER font. 

digital-fat-bwoy-slim-round

### Loader.qml changes:
- Use symbols for those checkbox descriptions that are easily self explanatory with a symbol and safe space.
- Add more descriptive ToolTips for first time users.
- Set delay of 600ms for ToolTips to appear when hoovering over them.
- Rearrange Items in the two Rows to not clip over the 640px viewport.

Visual change in the loader UI, left/before, right/after:

grafik

### CODE REVIEW

File: loader.qml [hunk 1]
```diff
@@ -102,41 +102,50 @@ ApplicationWindow {
...
+                    ToolTip.delay: 600
+                    ToolTip.text: qsTr("Switch to AmbientMode on black background")
                 }
 
                 CheckBox {
-                    id: ambientCheckBox
+                    id: halfSize
 
-                    text: qsTr("Dark")
+                    text: qsTr("320p")
```
Comment (FlorentRevest) [inline loader.qml:124]: I think 320px is easier to understand

File: loader.qml [hunk 2]
```diff
@@ -102,41 +102,50 @@ ApplicationWindow {
...
-
-                    text: qsTr("320x320")
-                }
-
-                CheckBox {
-                    id: twelveHourCheckBox
+                    id: ambientCheckBox
 
-                    text: qsTr("12h time")
+                    text: "\u2b24"
```
Comment (FlorentRevest) [inline loader.qml:115]: I wonder if a moon character wouldn’t be clearer that this is ambient mode.
I find the two round characters confusing, it sounds like they should be pretty much about the same thing where one is about the shape and one is about the mode.

File: digital-fat-bwoy-slim/usr/share/asteroid-launcher/watchfaces/digital-fat-bwoy-slim.qml [hunk 3]
```diff
@@ -0,0 +1,337 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.9
+import QtGraphicalEffects 1.12
+
+Item {
+
```
Comment (FlorentRevest) [inline digital-fat-bwoy-slim/usr/share/asteroid-launcher/watchfaces/digital-fat-bwoy-slim.qml:24]: Unnecessary empty line

**Comment by FlorentRevest** [comment]
Just a few things, overall it looks very good ;)

File: loader.qml (hunk 2 above)
Comment (eLtMosen) [reply loader.qml:115]: My original intention was to switch both round symbols from the Ambient and Round check boxes to squares when `Round` is unchecked.
Then imo  full black circle or square actually representing the current setting, would have made clearer that the background gets black in using that check box.
But i see you are even about AmbientMode being about more features then just a black background. Even in this simulation context, we hide and dim certain elements. Have not thought of that and indeed now agree that a moon sort of symbol would much better suggest a Mode change.

**Pull request merged**